examples/ksonnet/components/20-pod.jsonnet

```diff
@@ -76,25 +84,40 @@ local conf = {
     name: "sonobuoy",
 };
 
-local sonobuoyPod = local pod = k.core.v1.pod;
+
+{
+  objects(pullPolicy="Always", debug=false) ::
+    local opts = {
+      pullPolicy: pullPolicy,
+      debug: debug,
+    };
+
+    local myconf = conf(opts);
+    local sonobuoyPod =
```

Comment:
Is the open syntax kosher?  dangling =

Reply:
I found this really confusing at first too. I _think_ it's basically a "let", the `pod` is defined in the scope of defining sonobuoy.